### DISCUSSION TITLE
In osu! stable, there's times during a song, where confetti 'made up of stars' showers the screen, which doesn't seem to show up in the lazer build. Is it supposed to be no confetti during gamplay anymore?

### DISCUSSION THREAD

**Discussion by OO3300EE** [ppy/osu#27858] [Q&A] [answered]
I just downloaded the lazer version of osu, as I had taken a break from stable for some time before. I wanted to play osu but with some freshness added to it, so I got lazer. 
In lazer, whenever I played the song though, I felt as if something just felt off with the game, and after looking into it deeper, I soon figured out that songs never have that spray of stars being thrown out at the height of the song. It's something small that I still very much enjoyed, that would be a small quality-of-life thing to add, and funnily enough, _it's still sprayed on the home menu screen._ I was simply wondering if the spray of stars was a removed feature when moving away from stable? If it is going away, could we instead have a check-box in the settings, so that the ones who wants it still has access to it?

**Comment by cdwcgt** [comment]
![image](https://github.com/ppy/osu/assets/34775378/4e2f0402-1d0e-4b1e-929d-e88b55a7562d)
did you mean these star?
![image](https://github.com/ppy/osu/assets/34775378/c605e058-93f3-4420-b1a9-327f416614c5)
or these one? (lazer currently does not have)

**Comment by SupDos** [comment, marked as answer]
They haven't been removed on purpose, bringing them back is tracked in https://github.com/ppy/osu/issues/21091
